### ISSUE TITLE
Islamic Empire Administrative System

### ISSUE THREAD

**Issue by alltheatreides** [alltheatreides/AVE_MARIA_CK3#8]
## Province/Jund
Basic administrative district
- Led by an Amir/Wali named by the caliph/head of state
- Several level of control of the provincial Amir exist, at the central administrative level, the Amir has a short term of office preventing him from developing any power base. 
At the lowest level of control (Khidma ?) the Amir has an hereditary succession law.
- Provides taxes and no men (local troops for defense can be foreign (problems with the local population but higher quality and less revenue for the central government ; or local militia (rawabit), less expensive but low quality and loyal to the local interests.)
- The local qadi is named by the caliph/head of state and occupies the vanilla chancellor role (?). On his death an event is prompted to name a new qadi in the province. Qadi are drawn from learned men and cannot be former Amir.
- The longer an Amir stays in power the longer he can entrench his power and can refuse a dismissal by the caliph that can result in a revolt or simply declare independence.

## Tribal group
Duchy tier title representing a tribal contingent wether Bedouin or Turkic nomads. 
- Led by a tribal chief elected in an elective style succession weighed by martial qualities and kinship
- Provides no taxes but is always called to war automatically
- In the case of the Seljuks, they can declar war on their own and can drag their liege into some of their wars

## Thugur/Ajnad
Frontier districts forming the bulwark of frontier region with the Byzantines.
- Led by an Amir/Wali named by the caliph/head of state
- Should be able to raid against different religion ennemies.
- Can attract periodically ghazi soldiers through events, spawning some troops when war is declared against an ennemy of the faith or more rarely in peace time for a raid.
- Provides minimal taxes and no men (called to arms automatically in war with geographically close ennemies)

**Comment by thenunnceler** [comment]
Qadi are not really comparable to a steward, and court caplains/realm priests as represented in the game are a major anachronism. No mufti would serve in a role like that either. I would reccomend the Qadi be treated as the court chaplain, as he presided over religious law in his area. The rest of this is so cool!

**Comment by alltheatreides** [comment]
> Qadi are not really comparable to a steward, and court caplains/realm priests as represented in the game are a major anachronism. No mufti would serve in a role like that either. I would reccomend the Qadi be treated as the court chaplain, as he presided over religious law in his area. The rest of this is so cool!

In current WIP, I put Qadi in place of the Chancellor for the Amirs, and thought to convert the Court Chaplain into something more akin to Court Scholar or something, but it might be janky as it would not be the same for the Caliphs/Ruler.
Court Chaplain spot might be better, but for the Fatimids, the Court Chaplain spot would have been taken by the Fatimid Dawa so not sure.

**Comment by thenunnceler** [comment]
I'm not sure I follow. Da'wa isn't a title. Did you mean da'is or something?

**Comment by alltheatreides** [comment]
> I'm not sure I follow. Da'wa isn't a title. Did you mean da'is or something?

Yeah meant dai, the head of the fatimid proselytism.

**Comment by thenunnceler** [comment]
Right. That makes sense. Not sure where the qadi would fit then. They certainly didn't do diplomacy, but they also didn't proselytise, or lead armies or administer the realm per se. Could make them a role similar to a court physician and make them appear in events to do with criminal or shunned traits? Still leaves the "court mufti" thing super weird in other realms though.

**Comment by alltheatreides** [comment]
> Right. That makes sense. Not sure where the qadi would fit then. They certainly didn't do diplomacy, but they also didn't proselytise, or lead armies or administer the realm per se. Could make them a role similar to a court physician and make them appear in events to do with criminal or shunned traits? Still leaves the "court mufti" thing super weird in other realms though.

Regarding their tasks, if they are in the Court Chaplain spot, the Convert County task can be removed and replace with something like "Render Justice" or something along those lines.

EDIT: And the county conversion can happen through a different mean, through the proselytisc institutions

**Comment by thenunnceler** [comment]
That would make sense and be really cool. It would also make sense that the Fatimids would have available to them a more proactive approach in the form of a da'i.

So if I understand correctly that would be a Qadi in the court chaplain slot with no convert faith ability, except in the Fatimid realm, where it would be da'is, _with_ the convert faith ability. There would also be another means of conversion.

**Comment by alltheatreides** [comment]
> That would make sense and be really cool. It would also make sense that the Fatimids would have available to them a more proactive approach in the form of a da'i.
> 
> So if I understand correctly that would be a Qadi in the court chaplain slot with no convert faith ability, except in the Fatimid realm, where it would be da'is, _with_ the convert faith ability. There would also be another means of conversion.

Yes and no, a Qadi replacing the court chaplain yes, but I think the County Conversion ability could be removed from all muslim in general altogether and County Conversion happen differently, either through spontaneous local conversion or for the Fatimid, decisions/events by the head da'i.

**Comment by thenunnceler** [comment]
That makes sense.
